src/hooks/useActionHandlers/useActionHandlers.ts

```diff
@@ -11,6 +11,18 @@ export interface UseActionHandlersResult<T> {
     onKeyDown: React.KeyboardEventHandler<T>;
 }
 
+export function createOnKeyDownHandler<T>(callback?: AnyFunction) {
+    return (event: React.KeyboardEvent<T>) => {
+        if (
+            callback &&
+            [KeyCode.ENTER, KeyCode.SPACEBAR, KeyCode.SPACEBAR_OLD].includes(event.key)
+        ) {
+            // eslint-disable-next-line callback-return
```

Comment:
Lets explicitly return undefined to avoid suppressing es-lint.

Reply:
Removed this comment, it's just a warning